### PR TITLE
[SW-26] Menu is overlapping

### PR DESCRIPTION
**Pull request by svekukucska**
- Fixed responsive main menu on medium & small devices (made the menu toggle when an item is clicked)

### CODE REVIEW

File: src/components/MainMenu/index.js [hunk 1]
```diff
@@ -29,27 +30,38 @@ class MainMenu extends React.Component {
     });
   }
 
+  toggleMainMenu = () => {
```
Comment (spinenco-sv) [inline src/components/MainMenu/index.js]: let's rename this to something more relevant like: closeMainMenu?

File: src/components/MainMenu/index.js [hunk 2]
```diff
@@ -29,27 +30,38 @@ class MainMenu extends React.Component {
...
+
   render() {
     const {
       movies,
       dropdownOpen,
+      expanded,
     } = this.state;
-    const linkMovieList = (<span onClick={() => this.props.history.push('/movies')}> Star Wars Movies</span>);
+    const linkMovieList = (<span onClick={() => {
+      this.props.history.push('/movies');
```
Comment (spinenco-sv) [inline src/components/MainMenu/index.js]: Can you please move this into a separated function?

File: src/components/MainMenu/index.js [hunk 3]
```diff
@@ -29,27 +30,38 @@ class MainMenu extends React.Component {
...
+          <Navbar.Brand as={NavLink} to="/">
+            <img className="nav-logo" src={navLogo} alt="Star Wars nav logo"/>
           </Navbar.Brand>
-          <Navbar.Toggle aria-controls="responsive-navbar-nav" />
-          <Navbar.Collapse id="responsive-navbar-nav">
+          <Navbar.Toggle aria-controls="responsive-navbar-nav"/>
+          <Navbar.Collapse>
             <Nav className="m-auto">
-              <NavDropdown variant="tabs" title={linkMovieList} onMouseEnter={this.handleDropdownOpen} onMouseLeave={this.handleDropdownClose} show={dropdownOpen}>
+              <NavDropdown id={'myLinks'} variant="tabs" title={linkMovieList} onMouseEnter={this.handleDropdownOpen} onMouseLeave={this.handleDropdownClose} show={dropdownOpen}>
```
Comment (spinenco-sv) [inline src/components/MainMenu/index.js]: do you need this id?

File: src/components/MainMenu/index.js [hunk 4]
```diff
@@ -29,27 +30,38 @@ class MainMenu extends React.Component {
...
+      <Navbar collapseOnSelect expand="lg" bg="dark" variant="dark" fixed="top"
+        onToggle={() => this.setState({ expanded: !expanded })}
+        expanded={expanded}
+      >
+        <Container>
+          <Navbar.Brand as={NavLink} to="/">
+            <img className="nav-logo" src={navLogo} alt="Star Wars nav logo"/>
           </Navbar.Brand>
-          <Navbar.Toggle aria-controls="responsive-navbar-nav" />
-          <Navbar.Collapse id="responsive-navbar-nav">
```
Comment (spinenco-sv) [inline src/components/MainMenu/index.js:45]: you need this id for aria-controls, please add it back

File: src/components/MainMenu/index.js [hunk 5]
```diff
@@ -29,27 +30,38 @@ class MainMenu extends React.Component {
...
+          <Navbar.Toggle aria-controls="responsive-navbar-nav"/>
+          <Navbar.Collapse>
             <Nav className="m-auto">
-              <NavDropdown variant="tabs" title={linkMovieList} onMouseEnter={this.handleDropdownOpen} onMouseLeave={this.handleDropdownClose} show={dropdownOpen}>
+              <NavDropdown id={'myLinks'} variant="tabs" title={linkMovieList} onMouseEnter={this.handleDropdownOpen} onMouseLeave={this.handleDropdownClose} show={dropdownOpen}>
                 {movies && movies.map(({ episode_id, title }) => (
-                  <NavDropdown.Item as={NavLink} to={`/movies/${episode_id}`} key={episode_id}>{title}</NavDropdown.Item>
+                  <NavDropdown.Item as={NavLink} to={`/movies/${episode_id}`} key={episode_id} onClick={this.toggleMainMenu} >{title}</NavDropdown.Item>
```
Comment (spinenco-sv) [inline src/components/MainMenu/index.js]: Let's try to not have this type of lines, please add new lines here

**Review by spinenco-sv — CHANGES_REQUESTED** [review]

**Comment by svekukucska** [comment]
Requested changes have been implemented.

**Review by spinenco-sv — APPROVED** [review]

**Pull request merged**